docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Monitor Teleport Audit Events with Splunk
+title: Export Teleport Audit Events with Splunk
```

Comment:
The Splunk forwarder is exporting, but Splunk itself is collecting, indexing, etc.

Reply:
```suggestion
title: Export Teleport Audit Events to Splunk
```